wireapp/kalium
# feat: Order call participants list (AR-1232)

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

In a group call, if someone turn on his camera then we need to bring him to the top to be visible

### Solutions

We order the paarticipants list that will be displayed in a grid like this:
- self user comes first
- then participants with camera on sorted by named
- and finaly participants with camera off sorted by name


### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

Start a group call, if someone turnone his camera you should see him at the top of the list

### Notes (Optional)

Screen sharing will be supported in next PR

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
